test/built-ins/Array/prototype/concat/concat-array-likes-1.js

```diff
@@ -0,0 +1,677 @@
+// Copyright 2014 the V8 project authors. All rights reserved.
+// Use of this source code is governed by a BSD-style license that can be
+// found in the LICENSE file.
+
+// Flags: --harmony-arrays --harmony-classes
+
+/*---
+es6id: 22.1.3.1_3
+description: >
+    Array.prototype.concat: Symbol.isConcatSpreadable & array-likes
+---*/
+
+(function testArrayConcatArity() {
+  "use strict";
+  assert.deepEquals(1, Array.prototype.concat.length);
+})();
+
+
+(function testArrayConcatNoPrototype() {
+  "use strict";
+  assert.deepEquals(void 0, Array.prototype.concat.prototype);
+})();
+
+
+(function testArrayConcatDescriptor() {
+  "use strict";
```

Comment:
Enforcing "use strict" here seems bad since the test runner is usually used to signal that a test can run in both strict and non-strict. I doubt many of these depend on strict mode.